### ISSUE TITLE
View ship's orbital trajectory

### ISSUE THREAD

**Issue by s20dan** [pioneerspacesim/pioneer#985]
Now that we have some nifty fuel use going on for in-system flight, it finally becomes worthwhile to fly the most fuel-efficient route to the destination.
 To do that we need a visual representation of the player's orbital tradjectory, as well as the tradjectrories of any targets which we wish to rendezvous with (Planets and stars are already visualised)

 Here's some mspaint art ;)
First this is what the autopilot does:
![Imgur](http://i.imgur.com/opKak.png)

And this is what would be seen if the player were to perform a Hohmann transfer (As Brianetta mentions below this is not a true Hohmann transfer, but rather a higher energy transfer, although still using a lot less fuel than the autopilot)
![Imgur](http://i.imgur.com/38oSR.png)
The blue line would be the trajectory that the player's ship will take. 
 Basically this is just a way of visualising where the player is and where they will be, as you are always moving in space from one perspective or another, so its good to see where you will end up.

 It should be calculated based on the current reference point, and like set-speed mode ect should be able to be over-ridden with ctrl+click on a target.
 I imagine this would create some rather cool looking orbital configurations, like horseshoe orbits ect.

@Brianetta got any better diagrams? ;)
